### ISSUE TITLE
[Epic] Use Axe and Playwright for automated a11y testing

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14773]
## Description

Whilst using automated tooling is not a silver bullet for accessibility (a11y) testing, there are still a large number of [common issues](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) that can be highlighted/detected through automation that can be useful for investigation.

This is an issue to investigate and trial using [Axe](https://github.com/dequelabs/axe-core) for automated accessibility testing via [Playwright](https://playwright.dev/docs/accessibility-testing).

Rather than combine this into our regular CI integration tests (where issues might block a deployment), this could run these tests as a separate job on a clock schedule (e.g. once per day).

Combining Axe with Playwright is potentially useful as we can have tests interacting with the browser (clicking on elements, resizing the view port etc) before scanning for issues.

---

```[tasklist]
### Success Criteria
- [x] Add a11y tests for global components such as navigation, footer, subnav.
- [x] Add a11y tests for key, high-traffic pages for both desktop and mobile screen sizes.
- [x] Run a11y tests in automation on a daily basis, reporting failures in www-notify on Slack.
- [x] Create readable test reports, and surface those through automation.
- [x] Add documentation for a11y testing.
- [x] File a11y bugs.
- [x] Expand coverage.
- [ ] https://github.com/mozilla/bedrock/pull/15136
- [ ] https://github.com/mozilla/bedrock/pull/15462
```

```[tasklist]
### A11y bugs
- [ ] https://github.com/mozilla/bedrock/issues/14999
- [ ] https://github.com/mozilla/bedrock/issues/15000
- [ ] https://github.com/mozilla/bedrock/issues/15001
- [ ] https://github.com/mozilla/bedrock/issues/15002
- [ ] https://github.com/mozilla/bedrock/issues/15003
- [ ] https://github.com/mozilla/bedrock/issues/14770
- [ ] https://github.com/mozilla/bedrock/issues/15005
- [ ] https://github.com/mozilla/bedrock/issues/15060
- [ ] https://github.com/mozilla/bedrock/issues/15078
- [ ] https://github.com/mozilla/bedrock/issues/15137
- [ ] https://github.com/mozilla/bedrock/issues/15335
- [ ] https://github.com/mozilla/bedrock/issues/15336
- [ ] https://github.com/mozilla/bedrock/issues/15337
- [ ] https://github.com/mozilla/bedrock/issues/15338
- [ ] https://github.com/mozilla/bedrock/issues/15339
- [ ] https://github.com/mozilla/bedrock/issues/15341
- [ ] https://github.com/mozilla/bedrock/issues/15343
- [ ] https://github.com/mozilla/bedrock/issues/15344
- [ ] https://github.com/mozilla/bedrock/issues/15758
```
